Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/integrate-auth-system.md

![Custom login button on the Astronomer login screen](/img/docs/custom-oauth.png)

> You will also see the name you configured in `AUTH__OPENID_CONNECT__CUSTOM__DISPLAY_NAME` when authenticating via the Astro CLI.

Reviewer: ```suggestion
You can see the name you configured in `AUTH__OPENID_CONNECT__CUSTOM__DISPLAY_NAME` when authenticating using the Astro CLI.
```
